lib/logger/lib/logger/translator.ex

```diff
@@ -41,6 +41,11 @@ defmodule Logger.Translator do
   @doc """
   Built-in translation function.
   """
+  @spec translate(Logger.level(), Logger.level(), :format | :report, :logger.report()) ::
+          {:ok, iodata, keyword}
+          | {:ok, iodata}
+          | :skip
+          | :none
```

Comment:
I would also skip this one. It is already given by the callback, which we can't reuse easily, but it is good enough for me. :)